init_project.yml

```diff
@@ -1,6 +1,26 @@
 ---
 - import_playbook: create_project.yml
 - import_playbook: create_secrets.yml
-- import_playbook: create_config.yml
 - import_playbook: create_acme.yml
-- import_playbook: create_objects.yml
+
+- hosts: local
+  gather_facts: False
+  vars_files:
+    - "group_vars/customer/{{ customer }}.yml"
+    - "group_vars/env_type/{{ env_type }}.yml"
+
+  tasks:
+
+    - name: Print project details
+      debug: msg="Project name {{ project_name }} - domain {{ domain_name }}"
+
+    - name: Make sure base objects exist in OpenShift and are up-to-date
+      openshift_raw:
+        definition: "{{ lookup('template', 'templates/' + item + '.j2') | from_yaml }}"
+        state: present
+      with_items:
+        - "{{ openshift_volumes }}"
+        - "{{ openshift_endpoints }}"
+      when: item is not none
```

Comment:
I'm thinking: since routes are going to be fixed (previous, current, next) they should also be created in init. 
Could be part of @cchaudier's PR though WDYT?

Reply:
Good idea!